src/live_restore/live_restore_fs.c

```diff
@@ -1112,6 +1128,15 @@ __wt_live_restore_metadata_to_fh(
     if (!F_ISSET(S2C(session), WT_CONN_LIVE_RESTORE_FS))
         return (0);
 
+    /*
+     * Once we're in the clean up stage or later all data has been migrated across to the
+     * destination. There's no need for hole tracking and therefore nothing to import.
```

Comment:
Like on the bitmap PR, I suggest using some word other than `import` since that word may be confusing with the create-with-import API WT has. I don't know where the bitmap PR ended on that particular topic.

Reply:
ecf736108